packages/oas/src/analyzer/queries/openapi.ts

```diff
@@ -42,11 +42,21 @@ export async function circularRefs(definition: OASDocument) {
   return results;
 }
 
+/**
+ * Determine if a given API definition utilizes common parameters.
+ *
+ * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#path-item-object}
+ * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#path-item-object}
+ */
+export function commonParameters(definition: OASDocument) {
+  return query(['$..paths[*].parameters'], definition).map(res => refizePointer(res.pointer));
```

Comment:
i believe the answer is "no" but just to double check — are there any other places in the tree where a parameter can be defined (other than at the path- and operation-level)?